### DISCUSSION TITLE
example outputs?

### DISCUSSION THREAD

**Discussion by doulemint** [CoBrALab/RABIES#257] [Q&A] [answered]
I wonder whether rabies offers an official example outputs for reference of standard outputs?

**Comment by Gab-D-G** [comment, marked as answer]
We provide a test dataset which should run smoothly https://zenodo.org/record/3937697#.Y7b_r9LMI5k, but unfortunately we don't currently provide the expected outputs. Ideally, we will put together a complete tutorial with a step-by-step guide and example outputs. For now the documentation includes a set of example outputs from QC (not from this test dataset) in the preprocessing quality control webpage https://rabies.readthedocs.io/en/stable/quality_control.html. These outputs represent successful registration steps.
